languages/python/exercises/concept/bools/.meta/config.json

```
@@ -0,0 +1,16 @@
+{
+  "authors": [
+    {
+      "github_username": "neenjaw",
+      "exercism_username": "neenjaw"
+    }
+  ],
+  "contributors": [
+    {
+      "github_username": "cmccandless",
+      "exercism_username": "cmccandless"
+    }
+  ],
+  "forked_from": ["elixir/booleans"],
+  "language_versions": ">=3.8"
```

Currently, we support Python 3.5 and higher. I don't see any features being used in the code here that isn't supported in 3.5.

```suggestion
  "language_versions": ">=3.5"
```

---

Sure, I didn't know, I just put what I was using :P